topics/fair/tutorials/data-management-peatlands/tutorial.md

```diff
@@ -0,0 +1,118 @@
+
+| DMP component  | What to include|
+| ------------- | ------------- |
+| Descriptions of the data  |    &bull; Project details <br> &bull; Types of data <br> collected and generated <br> &bull; Origin of data <br> &bull; Format and size of data |
+|Data collection/generation | &bull; Data collection methods <br> &bull; Data quality standards| 
+|Data management, documentation, and curation | &bull; Storage and accessibility <br> &bull; Metadata standards and documentation <br> &bull; Curation and preservation |  
+|Data sharing and access | &bull; Where the data will be shared <br> &bull; When will the data be available |
+|Data security (where relevant)|&bull; Risks to data security<br> &bull; GDPR considerations|
+|Capabilities| &bull; Evaluation of institutional capabilities to preserve and manage data|
+|Maintaining and implementing the Data Management Plan|&bull; Actions to maintain and deliver the DMP |
+|Environmental considerations|&bull; Energy cost considerations related to data management|
+|Responsibilities|&bull; Agreement on the responsibilities of named individuals for data management|
+|Relevant institutional, departmental or study policies on data sharing and data security| &bull; Review of institutional policies for data management|
+
+## Hands-on: Create a DMP for peatland research
+* Download the DMP template for [Peatland Research and Monitoring](https://zenodo.org/records/11185774).
+* Fill in the DMP template using the prompt questions.
+* Review your DMP with your team and collaborators.
+
+# FAIR data management solutions
+
+After you have filled in the DMP template, review how you can make your DMP FAIR.
+
+There are several ways to set up FAIR (Findable, Accessible, Interoperable, Reusable) data management plans (DMPs) 
+
+* Findable (F): Data description and collection or reuse of existing data.  Using metadata and community recognised standards for curation that enable someone to find the data.
+* Accessible (A): Standardised authentication or authorisation (e.g. HTTP, HTTPS) for open and restricted data access.
+* Interoperable (I): Documentation and data quality, and community standards for data exchange (e.g. tab-delimited text).
+* Reusable (R): Storage and backup supported by legal and ethical requirements, as well as data release policies.
+
+More information is available in the [FAIR data management tutorial](https://training.galaxyproject.org/training-material/topics/fair/tutorials/data-management/tutorial.html)
+
+# Conclusion
+
+Creating a DMP will allow you and your institution to make the most of your peatland data. This introduction to DMP aims to highlight the usefulness of data management for peatland research and monitoring.
+
+# References
+*Wilkinson, M. D., M. Dumontier, I. J. J. Aalbersberg, G. Appleton, M. Axton et al., 2016 The FAIR Guiding Principles for scientific data management and stewardship. Scientific data 3: 1–9. [10.1038/sdata.2016.18](https://www.nature.com/articles/sdata201618)
```

Comment:
Could you create a tutorial.bib file with this citation? The References section will then be automatically added to the end of the tutorial.